C/regression-tests/regression-tests/gpio/main.c

```diff
@@ -123,4 +123,14 @@ int main()
     print_str("DDRB_REG: ");
     print_num(DDRB_REG);
     print_str("\n");
+    print_num_binary(0);
+    print_str("\n");
+    print_num_binary(255);
+    print_str("\n");
+    print_num_binary(16);
```

Comment:
I think it will be good if the number is prefixed by `0b` as indication for binary numbers